docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -160,7 +202,12 @@ A list of outputs (i.e. forwarding targets) for the Elastic Serverless Forwarder
 
 `inputs.[].outputs.[].type`:
 
-The type of the forwarding target output (currently only `elasticsearch` supported).
+The type of the forwarding target output. Currently only the following outputs are supported:
+
+ * `elasticsearch`
+ * `logstash`
```

Comment:
```suggestion
 * preview:[] `logstash`
```
I think you might have applied this before I edited it to include the preview admonition.